### ISSUE TITLE
"Look at" vs "look in"

### ISSUE THREAD

**Issue by SaphiraSkies** [CharlesBeise/ScaryAttic#89]
As I've been testing the game, I find myself typing "look in" instead of "look at" a few times. Although it isn't strictly necessary, it might be nice to make the phrase "look in" equivalent to "look at". (Example: "look in moving boxes" or "look in tub")

**Comment by CharlesBeise** [comment]
Yeah, that would make the game feel much better. I made some tweaks to allow for this:

https://github.com/CharlesBeise/ScaryAttic/pull/91
